File: backend/poster-service/Sources/App/DTOs/SendEmailDTO.swift
public struct SendEmailDTO: Codable {
    public var receiver: String
    public var subject: String
    public var message: String?
    public var template: String?
    public var templateData: [String: String]?

    public init(
        receiver: String,
        subject: String,
        message: String? = nil,
        template: String? = nil,
        templateData: [String: String]? = nil
    ) {
        self.receiver = receiver
        self.subject = subject
        self.message = message
        self.template = template
        self.templateData = templateData
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/NotificationsServiceDTOs/SendEmailDTO.kt
package net.ipv64.kivop.dtos.NotificationsServiceDTOs

import java.util.UUID

import java.time.LocalDateTime

data class SendEmailDTO (
    var receiver : String,
    var subject : String,
    var message : String?,
    var template : String?,
    var templateData : Map<String, String>?
)
